Mathlib/MeasureTheory/Integral/Indicator.lean

```diff
@@ -5,6 +5,7 @@ Authors: Kalle Kytölä
 -/
 import Mathlib.MeasureTheory.Integral.Lebesgue
 import Mathlib.MeasureTheory.Constructions.BorelSpace.Metrizable
+import Mathlib.Topology.IndicatorConstPointwise
```

Comment:
This import is light. The new file is less than 75 lines and should not import anything new transitively.